### PR TITLE
docs: update grammar in local setup guide

### PR DESCRIPTION
**Pull request by iamrajiv**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

### CODE REVIEW

File: docs/how-to-setup-freecodecamp-locally.md [hunk 1]
```diff
@@ -178,11 +178,11 @@ cp sample.env .env
 copy sample.env .env
 ```
 
-The keys in the `.env` file are *not* required to be changed to run the app locally. You can leave the default values copied over from `sample.env` as-is.
+The keys in the `.env` file is *not* required to be changed to run the app locally. You can leave the default values copied over from `sample.env` as-is.
```
Comment (RandellDawson) [inline docs/how-to-setup-freecodecamp-locally.md]: Please revert this line as the verb "are" is for the word "keys" and not "file".

File: docs/how-to-setup-freecodecamp-locally.md [hunk 2]
```diff
@@ -178,11 +178,11 @@ cp sample.env .env
...
 
-Keep in mind if you want to use additional services, you'll have to acquire your own API keys for those services and edit the entries accordingly in the `.env` file.
```
Comment (RandellDawson) [inline docs/how-to-setup-freecodecamp-locally.md]: No reason to change this.

File: docs/how-to-setup-freecodecamp-locally.md [hunk 3]
```diff
@@ -153,7 +153,7 @@ npm -v
 If you have a different version, please install the recommended version. We can only support installation issues for recommended versions.
 
 Windows users:
-> Make sure the command line tool you use (e.g., cmd, PowerShell, Git Bash for Windows, WSL) has the correct user privileges. If possible, you should launch the tool with Administrator's privilege. On Windows you should be able to launch as administrator by right-clicking the application and selecting `Launch as an Administrator`.
+> Make sure the command-line tool you use (e.g., cmd, PowerShell, Git Bash for Windows, WSL) has the correct user privileges. If possible, you should launch the tool with the Administrator's privilege. On Windows, you should be able to launch as administrator by right-clicking the application and selecting `Launch as an Administrator`.
```
Comment (RandellDawson) [inline docs/how-to-setup-freecodecamp-locally.md]: No reason to add the word "the" here.

File: docs/how-to-setup-freecodecamp-locally.md [hunk 4]
```diff
@@ -6,20 +6,20 @@
 # Set up freeCodeCamp locally
 Follow these guidelines for getting freeCodeCamp locally on your system. This is highly recommended if you want to contribute regularly.
 
-Some of the contribution workflows, like previewing pages for the guide or the coding challenges, debugging and fixing bugs in codebase, requires you to have freeCodeCamp running locally.
+Some of the contribution workflows, like previewing pages for the guide or the coding challenges, debugging and fixing bugs in the codebase, requires you to have freeCodeCamp running locally.
 
 ## Fork the repository on GitHub
 ['Forking'](https://help.github.com/articles/about-forks/) is a step where you get your own copy of freeCodeCamp's main repository (a.k.a _repo_) on GitHub.
 
-This is essential as it allows you to work on your own copy of freeCodeCamp on GitHub, or to download (clone) your repository to work on locally. Later, you will be able to request changes to be pulled into the main repository from your fork via a pull request.
```
Comment (RandellDawson) [inline docs/how-to-setup-freecodecamp-locally.md]: Please add the word "own" back in.

File: docs/how-to-setup-freecodecamp-locally.md [hunk 5]
```diff
@@ -328,9 +328,9 @@ Follow these steps:
 
 2. Sync the latest changes from the freeCodeCamp upstream `master` branch to your local master branch
 
-    **Note:** If you have any outstanding Pull Request that you made from the `master` branch of your fork, you will lose them at the end of this step. You should ensure your pull request is merged by a moderator prior to performing this step. To avoid this scenario, you should *always* work on a branch separate from master.
+    **Note:** If you have any outstanding Pull Request that you made from the `master` branch of your fork, you will lose them at the end of this step. You should ensure your pull request is merged by a moderator before performing this step. To avoid this scenario, you should *always* work on a branch separate from master.
 
-    This step **will sync the latest changes** from the main repository of freeCodeCamp. It is important that you rebase your branch on top of the latest `upstream/master` as often as possible to avoid conflicts later.
+    This step **will sync the latest changes** from the main repository of freeCodeCamp. You must rebase your branch on top of the latest `upstream/master` as often as possible to avoid conflicts later.
```
Comment (RandellDawson) [inline docs/how-to-setup-freecodecamp-locally.md]: This is not a case of "must do" here.  This should be reverted.

**Review by RandellDawson — CHANGES_REQUESTED** [review]
@iamrajiv See my suggested changes.

**Review by raisedadead — APPROVED** [review]

**Pull request merged**